src/functionalTest/resources/features/F-017/F-017.feature

```diff
@@ -0,0 +1,88 @@
+    Given an appropriate test context as detailed in the test data source,
+    And a user [Befta1 - who invokes the API],
+    When a request is prepared with appropriate values,
+    And the request [has roleType Organisation, roleCategory Professional, roleName Role1 and process value is not set to 'professional-organisational-role-mapping'],
+    And it is submitted to call the [Create Role Assignments] operation of [Role Assignments Service],
+    Then a negative response is received,
+    And the response has all other details as expected
+
+  @S-254
+  Scenario: Invoking POST role-assignments api with request to add a new role assignment with not valid classification
+    Given an appropriate test context as detailed in the test data source,
+    And a user [Befta1 - who invokes the API],
+    When a request is prepared with appropriate values,
+    And the request [has roleType Organisation, roleCategory Professional, roleName Role1 and Classification is not set to Restricted],
+    And it is submitted to call the [Create Role Assignments] operation of [Role Assignments Service],
+    Then a negative response is received,
+    And the response has all other details as expected
+
+  @S-255
+  Scenario: Invoking POST role-assignments api with request to add a new role assignment with not valid grantType
+    Given an appropriate test context as detailed in the test data source,
+    And a user [Befta1 - who invokes the API],
+    When a request is prepared with appropriate values,
+    And the request [has roleType Organisation, roleCategory Professional, roleName Role1 and grantType not set to STANDARD],
+    And it is submitted to call the [Create Role Assignments] operation of [Role Assignments Service],
+    Then a negative response is received,
+    And the response has all other details as expected
+
+  @S-256
+  Scenario: Invoking GET role assignment with valid actorId and returns 200 OK
+    Given an appropriate test context as detailed in the test data source,
+    And a user [Befta1 - who invokes the API],
+    And a successful call [to create org role assignments for actors & requester] as in [S-256_Org_Role_Creation],
+    When a request is prepared with appropriate values,
+    And the request [has a actor id passed that does not exists],
+    And it is submitted to call the [Create Role Assignments] operation of [Role Assignments Service],
+    Then a positive response is received,
+    And the response has all other details as expected
+
+  @S-257
+  Scenario: Invoking GET role assignment with an invalid actorId and returns 200 OK
+    Given an appropriate test context as detailed in the test data source,
+    And a user [Befta1 - who invokes the API],
+    And a successful call [to create org role assignments for actors & requester] as in [S-256_Org_Role_Creation],
```

Comment:
If we are testing for an invalid (non existing user), we no need to create an ORG role.
You can use the actorId some like this : cf00ea00-00c0-0000-b0df-e0000d00b000

Reply:
Have updated as per review comments and removed the org role creation step